### ISSUE TITLE
Docker Image includes `.git` folder

### ISSUE THREAD

**Issue by ytausch** [regro/cf-scripts#2876]
Our Docker image currently contains a `.git` folder, which is not needed in the image. Probably there are even more things towards a minimal image, which will also be faster to build.

**Comment by beckermr** [comment]
How much space is the .git folder? My guess is that there are bigger things to do than that?

**Comment by ytausch** [comment]
Yes, probably. I just wanted to point out the git folder because it came to my attention as something that should clearly not be in the image. The git folder also contains a git token provided by GitHub which, I confirmed, has no security relevance for us.

**Comment by jakirkham** [comment]
Should be easy to add to `.dockerignore`
